sidecar/internal/csiaddonsnode/csiaddonsnode.go

```diff
@@ -58,6 +60,16 @@ type Manager struct {
 	// Endpoint is the location where the sidecar receives connections on
 	// from the CSI-Addons Controller.
 	Endpoint string
+
+	// PodName is the (unique) name of the Pod that contains this sidecar.
+	PodName string
+
+	// PodNamespaec is the Kubernetes Namespace where the Pod with this
```

Comment:
`PodNamespaec` -> `PodNamespace`

Reply:
Nit

Reply:
Ah, I thought I addressed it! Done now, thanks!